addon/components/bourbon-select-field.js

```diff
@@ -44,39 +45,46 @@ export default Component.extend(SelectMixin, {
   }),
 
   focusOut() {
+    this.set('activeOption', null);
     this.set('showList', false);
   },
 
   mouseDown() {
+    this.set('activeOption', null);
     this.set('showList', !this.get('showList'));
   },
 
   findValueObject(valueString) {
-    for (var optIndex in this.get('content')) {
-      if (this.get('content')[optIndex].value === valueString) {
+    for (var optIndex of this.get('content')) {
```

Comment:
did the type of `this.get('content')` change? not sure why this change from `in` to `of`

Reply:
not sure what happened here.  i switched it back to in.